webpack/stories/index.js

```diff
@@ -8,6 +8,7 @@ import StatisticsChartsList from
 '../assets/javascripts/react_app/components/charts/StatisticsChartsList';
 import StatisticsChartBox from
 '../assets/javascripts/react_app/components/charts/StatisticsChartBox';
+import withDataProps from './data/statistics/WithDataProps';
```

Comment:
should we call that with a different name (e.g. less generic than withDataProps?)